src/assets/style.css

```diff
@@ -49,9 +49,3 @@
   left: 0;
 }
 
-.playkit-subtitles > div > div {
-  -webkit-transition: 0.3s ease-in-out;
-  -moz-transition: 0.3s ease-in-out;
```

Comment:
@odedhutzler 
From what remember Safari transition is only the `-moz-` transition (needs to verify that), so why remove all?

Reply:
1. Safari is webkit, moz is for mozilla (Firefox). Removing everything because i don't understand why we handle this here - i think it should be handled in the playkit-js-ui. more than that - i think the content of this file should move to the playkit-js-ui.

2. of course.

Reply:
Great, this is why I wrote that need to verify, so need to remove the webkit.
The core is in charge of displaying the captions and provide api to change their styles, the ui is in charge to use this api.

Reply:
ah - so this should be checked without the ui, to see this change doesn't mess the default captions.